gravity-forms/gw-submit-to-access.php

```diff
@@ -355,7 +356,7 @@ public function add_submitted_form( $form ) {
 			if ( $this->_args['enable_user_meta'] && is_user_logged_in() ) {
 				update_user_meta( get_current_user_id(), 'gwsa_submitted_forms', $submitted_forms );
 			} else {
-				$expiration = $this->_args['is_persistent'] ? strtotime( '+1 year' ) : null;
+				$expiration = $this->_args['is_persistent'] ? strtotime( '+1 year' ) : $this->_args['cookie_expiration'];
```

Comment:
Should this be something like the following?

```suggestion
				$expiration = $this->_args['is_persistent'] ? rgar( $this->_args, 'cookie_expiration', strtotime( '+1 year' ) ) : null;
```

If it's _not_ persistent, it should always be null, right?

Reply:
That wasn't exactly the original intent of the "is_persistent" parameter but I like making it so moving forward.

Reply:
@spivurno Can you elaborate? Isn't the idea that if it's not persistent it's only for the session?